src/utils/helpers.ts

```diff
@@ -0,0 +1,34 @@
+import { Errors } from '../types/Errors';
+import { FilterType } from '../types/FilterType';
+import { Todo } from '../types/Todo';
+
+export const filterTodos = (todos: Todo[], filterType: FilterType) => {
+  return todos.filter(todo => {
+    switch (filterType) {
+      case FilterType.All:
+        return true;
+      case FilterType.Active:
+        return !todo.completed;
+      case FilterType.Completed:
+        return todo.completed;
+      default:
+        return true;
+    }
+  });
+};
+
+export const unsetError = (
+  setterFunc: (error: Errors | null) => void,
+  errorType: Errors | null,
+  delay: number,
+) => {
+  if (delay === undefined) {
```

Comment:
```suggestion
  if (!delay) {
```

Reply:
> 

Not fixed, and actually you don't use this function in your code, so get rid of it